modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm

```diff
@@ -43,6 +43,13 @@
 	rpg_title = "Battle Cleric"
 	job_flags = JOB_ANNOUNCE_ARRIVAL | JOB_CREW_MANIFEST | JOB_EQUIP_RANK | JOB_CREW_MEMBER | JOB_NEW_PLAYER_JOINABLE | JOB_REOPEN_ON_ROUNDSTART_LOSS | JOB_ASSIGN_QUIRKS | JOB_CAN_BE_INTERN
 
+	veteran_only = TRUE
+
+/datum/job/security_medic/after_spawn(mob/living/spawned, client/player_client)
+	. = ..()
+	to_chat(player_client, span_doyourjobidiot("As a Security Medic your primary focus is to keep prisoners and officers alive. This means you should not be focusing on civilian type medical duties, leave that to the paramedic and medical. Do not make a mini-medical center within security, always take people that need in-depth medical attention to medbay. If you are in medbay, you report to the CMO!"))
```

Comment:
isn't there a config file for doing exactly this?

Reply:
whewre tho

Reply:
`policy.json`

Reply:
done